.github/pull_request_template.md

```diff
@@ -5,10 +5,15 @@ What GitHub Desktop issue does this PR address? (for example, #1234)
 Closes #[issue number]
 
 ## Description
-
+<!--
+What changes did you make to the code?
+-->
 -
 
 ### Screenshots
+<!--
+Insert your screenshots here
+-->
```

Comment:
I think the comment below on 19 is enough to describe this section.
```suggestion
```

Reply:
K, will do